source/includes/common-mc-admin-config.rst

```diff
@@ -0,0 +1,978 @@
+.. Descriptions for AMQP bucket notification configurations.
+   Used in the following files:
+   - /source/reference/minio-server/minio-server.rst
+   - /source/reference/minio-cli/minio-mc-admin/mc-admin-config.rst
+
+.. start-minio-notify-amqp-enable
+
+Specify ``on`` to enable publishing bucket notifications to an AMQP endpoint.
+
+Defaults to ``off``.
+
+.. end-minio-notify-amqp-enable
+
+
+.. start-minio-notify-amqp-url
+
+Specify the AMQP server endpoint to which MinIO publishes bucket events.
+For example, ``amqp://myuser:mypassword@localhost:5672``.
+
+.. end-minio-notify-amqp-url
+
+
+.. start-minio-notify-amqp-exchange
+
+Specify the name of the AMQP exchange to use.
+
+.. end-minio-notify-amqp-exchange
+
+
+.. start-minio-notify-amqp-exchange-type
+
+Specify the type of the AMQP exchange.
+
+.. end-minio-notify-amqp-exchange-type
+
+
+.. start-minio-notify-amqp-routing-key
+
+Specify the routing key for publishing events.
+
+.. end-minio-notify-amqp-routing-key
+
+
+.. start-minio-notify-amqp-mandatory
+
+Specify ``off`` to ignore undelivered messages errors. Defaults to ``on``.
+
+.. end-minio-notify-amqp-mandatory
+
+
+.. start-minio-notify-amqp-durable
+
+Specify ``on`` to persist the message queue across broker restarts. Defaults to
+'off'.
+
+.. end-minio-notify-amqp-durable
+
+
+.. start-minio-notify-amqp-no-wait
+
+Specify ``on`` to enable non-blocking message delivery. Defaults to 'off'.
+
+.. end-minio-notify-amqp-no-wait
+
+
+.. start-minio-notify-amqp-internal
+
+Specify ``on`` to use the exchange only if it is bound to other exchanges.
```

Comment:
@poornas @vadmeste this explanation is coming straight from the CLI - I think we could use a bit more clarity here, and I couldn't find much within AMQP as to what 'bound to other exchanges' signified.

Reply:
> @poornas @vadmeste this explanation is coming straight from the CLI - I think we could use a bit more clarity here, and I couldn't find much within AMQP as to what 'bound to other exchanges' signified.

Well as I saw, exchanges are a known term in AMQP, but I am not sure why would someone need this option. Better to ping the developer who sent AMQP support

Reply:
I'll dig around.

Reply:
Ah - https://www.rabbitmq.com/e2e.html. Not sure how I missed that.